packages/uniforms/__suites__/ListField.tsx

```diff
@@ -116,4 +183,46 @@ export function testListField(
     expect(onChange).toHaveBeenCalledTimes(1);
     expect(onChange).toHaveBeenLastCalledWith('x', [undefined]);
   });
+
+  if (testError) {
+    test('<ListField> - renders correct error text (specified)', async () => {
+      const error = new Error();
+
+      render(
+        <ListField
+          name="x"
+          error={error}
+          errorMessage="Error"
+          showInlineError
+        />,
+        {
+          x: Array,
+          'x.$': String,
+        },
+      );
+
+      expect(screen.getByText(/^Error$/)).toBeInTheDocument();
+    });
+  }
+
+  if (disableInlineError) {
+    test('<ListField> - renders correct error text (specified)', async () => {
```

Comment:
Is the name of this test OK? It's no different than the above one but the body is different (e.g. `showInlineError={false}`.